app/controllers/authenticate_controller.rb

```diff
@@ -207,9 +207,9 @@ def status_failure_response(error)
     status_code = case error
                   when Errors::Authentication::Security::RoleNotAuthorizedOnResource
                     :forbidden
-                  when Errors::Authentication::StatusNotImplemented
+                  when Errors::Authentication::StatusNotSupported
                     :not_implemented
-                  when Errors::Authentication::AuthenticatorNotFound
+                  when Errors::Authentication::AuthenticatorNotSupported
```

Comment:
same goes for here
See above comment [here](https://github.com/cyberark/conjur/pull/1784/files#r478382595)